cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java

```diff
@@ -83,6 +83,13 @@ public final class DataprocUtils {
   // This property needs to be overridden in cdap-site.
   public static final String TROUBLESHOOTING_DOCS_URL_DEFAULT = "";
 
+  // resources required by Runtime Job (io.cdap.cdap.runtime.spi.runtimejob.RuntimeJob)
+  // that will be running on driver pool nodes
+  public static final String DRIVER_MEMORY_MB = "driverMemoryMB";
+  public static final String DRIVER_MEMORY_MB_DEFAULT = "2048";
```

Comment:
Good to explain why these defaults were chosen.

Reply:
Discussed offline, these defaults will be tuned after performance tests.